### PR TITLE
first pull req

### PR DESCRIPTION
**Pull request by vijaypurohit**

### CODE REVIEW

**Comment by vijaypurohit** [comment]
nice

File: Dynamic-Programming/GFG_SubsetSum.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// { Driver Code Starts
+//Initial template for C++
+
```
Comment (prateekshyap) [inline Dynamic-Programming/GFG_SubsetSum.cpp:7]: Please add geeksforgeeks practice link in GFG_SubsetSum.cpp

Comment (vijaypurohit) [reply Dynamic-Programming/GFG_SubsetSum.cpp:7]: ohh really, i might have forgotten.

**Review by prateekshyap — APPROVED** [review]
Will be approved

**Pull request merged**